main.go

```diff
@@ -10,15 +11,18 @@ import (
 	"strings"
 	"time"
 
+	"./trailhead"
```

Comment:
You'll want to switch this import statement back to `"github.com/meruff/go-trailhead-leaderboard-api/trailhead"` actually, as that's how Golang prefers importing packages.

Reply:
Maybe I don't understand the way Go works enough but it seems to me that this would be preferred when the referenced package is not in the same repository. With this approach then you would have to commit/push changes to the trailhead folder first before you could make changes to the main file. In this case, I would need to create a PR for you to pull into your repo before the main file on my machine would even work